packages/core/src/services/essentials/gradient-utils.ts

```diff
@@ -0,0 +1,75 @@
+// Internal Imports
+import { Service } from "../service";
+import { DOMUtils } from "../";
+
+export class GradientUtils extends Service {
+	static appendLinearGradient(svg, id, x1, x2, y1, y2, stopAttrList) {
```

Comment:
can this be a `configs` flag?
```suggestion
	static appendLinearGradient(configs) {
```

Reply:
this way where you're calling `appendLinearGradient` we won't just see 6 unnamed parameters, instead we'll see

```js
appendLinearGradient({
  svg: "blah",
  id: "blah",
  ............................
})
```

much more readable

Reply:
and we should just rename `stopAttrList` to `stops`.

They're not really `attrs` that are being passed in, rather they are actual stop definitions that we use to apply attrs to our element.